tests/lambda_handlers_tests/test_convertHandler.py

```diff
@@ -1,12 +1,49 @@
 from __future__ import absolute_import, unicode_literals, print_function
 import mock
 from unittest import TestCase
+from requests import Response
 from libraries.lambda_handlers.convert_handler import ConvertHandler
 from libraries.converters.md2html_converter import Md2HtmlConverter
 from libraries.converters.usfm2html_converter import Usfm2HtmlConverter
 
 
 class TestConvertHandler(TestCase):
+
```

Comment:
Move to test_converter.py so we aren't testing the lambda handler function once the code is moved to converter.py.